nautobot/extras/jobs.py

```diff
@@ -977,7 +977,7 @@ def _get_job_source_paths():
             else:
                 logger.warning(f"Git repository {repository_record} is configured to provide jobs, but none are found!")
 
-        # TODO: when a Git repo is deleted or its slug is changed, we update the local filesystem
+        # TODO(Glenn): when a Git repo is deleted or its slug is changed, we update the local filesystem
```

Comment:
What clarification are you looking for here? I think the comment is pretty clear.